funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -280,6 +275,12 @@ def start_endpoint(
                     "process. The endpoint will not work until it is successfully "
                     "registered."
                 )
+            elif (
+                e.http_status == EndpointLockedError.http_status_code
+                or e.http_status == EndpointInUseError.http_status_code
+            ):
+                log.warning(f"Endpoint registration blocked. {e.message}")
+                exit(-1)
```

Comment:
Wow; the fact that we missed this until now implies that this section is not covered by a UT at all.  And ... in fact, this does not appear to be covered:

<img src="https://user-images.githubusercontent.com/99676336/205973751-9b53ce74-87e8-42aa-af84-1ddf548d8ceb.png" width="500"/>

Ugh.

Reply:
I've created #985 so as to properly UT at least one of those branches, while addressing the error your observed.  That'll make this PR dead simple.